test/integration/users.test.js

```diff
@@ -585,6 +574,38 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users/?id", function () {
+    it("Should return given user by id", function (done) {
+      chai
+        .request(app)
+        .get(`/users/?id=${userId}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("User returned successfully!");
+          expect(res.body.user).to.be.a("object");
```

Comment:
shouldn't we try to get the data from db and check weather returned user id matches the query param id passed?

Reply:
should we fetch all users and then check if user exist, with id passed in query param? Is this we should do.

Reply:
Fetch the user using the userId.

Reply:
We are doing this already.

Reply:
@RitikJaiswal75 has asked to fetch the data from db, we are only getting hardcoded values.

Reply:
How do i check that the data returned has the same user id as the user id passed in query params

Reply:
We are not querying the db here, just the API is being called.

`await userQuery.fetchUser({ userId: id });` this is how we query database

Reply:
modified test as required.